op-service/log/cli.go

```diff
@@ -93,10 +93,10 @@ func ReadLocalCLIConfig(ctx *cli.Context) CLIConfig {
 
 func ReadCLIConfig(ctx *cli.Context) CLIConfig {
```

Comment:
Can we deprecate `ReadLocalCLIConfig`? Do we have any other spots where we had a difference between local & global?

Reply:
will remove `ReadLocalCLIConfig`, there are not many usages of it, and it shouldn't make a difference anymore.